### DISCUSSION TITLE
Kann man den Typ mal endlich sperren?

### DISCUSSION THREAD

**Discussion by stupidhackers** [community/community#134069] [Repositories] [answered]
### Select Topic Area

General

### Body

Hallo,

der Typ: https://github.com/sindresorhus/got

verbreitet täglich per REFERRER-Spam für sein BOT Summaly!!!

Kann man das Konto https://github.com/sindresorhus/got mal ENDLICH SPERREN?

Hier sein Referrer-Spam, kommt täglich auf über 14 deutschen und englischen Websites, der Name der betroffenen Websites wurde zum Schutz der Besitzer umbenannt:

161.129.34.20 www.website-1.de - - [30/Jul/2024:10:02:33 +0200] "GET /solaranlagen/balkonkraftwerk/balkonkraftwerk-600w-upgrade-800w.php HTTP/1.0" 200 55556 "-" "SummalyBot/5.0.3"
161.129.34.20 www.website-1.de - - [30/Jul/2024:10:02:36 +0200] "HEAD /wp-content/uploads/2022/01/Icon-Balkonkraftwerk.jpg HTTP/1.0" 200 - "-" "got (https://github.com/sindresorhus/got)"
ASN 	AS138195 - MOACK.Co.LTD
Range 	161.129.34.0/24
NetRange:       161.129.34.0 - 161.129.34.255
Seoul, South Korea

79.133.252.100 web-site1.com - - [28/Jul/2024:08:40:23 +0200] "GET /balkon-solaranlage/balkonkraftwerk/balkonkraftwerke-fuer-Mieter HTTP/1.0" 200 49869 "-" "SummalyBot/2.7.0"
79.133.252.100 web-site1.com - - [28/Jul/2024:08:40:25 +0200] "HEAD /wp-content/uploads/2022/08/Icon-Balkonkraftwerk.jpg HTTP/1.0" 302 - "-" "got (https ://github.com/sindresorhus/got)"
ASN 	AS21412 - UAB "Cgates" - ISP
Range 	79.133.224.0/19
inetnum:        79.133.224.0 - 79.133.255.255
Vilnius, Lithuania

76.67.133.136 www.web-site2.de - - [28/Jul/2024:08:44:55 +0200] "GET /balkonkraftwerk-2/balkonkraftwerk-1000w-maximale-energie-von-ihrem-balkon.php HTTP/1.0" 200 56302 "-" "SummalyBot/2.7.0"
76.67.133.136 www.web-site1.de - - [28/Jul/2024:08:44:56 +0200] "HEAD /wp-content/uploads/2022/01/Icon-Balkonkraftwerk.jpg HTTP/1.0" 200 - "-" "got (https ://github.com/sindresorhus/got)"
ASN 	AS577 - Bell Canada - ISP
Range 	76.67.0.0/16
NetRange:       76.67.224.0 - 76.67.255.255
Ottawa, Canada

**Comment by queenofcorgis** [comment, marked as answer]
Hi @stupidhackers 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!

Please note for future posts: 
We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting.
